test/suites/integration/bsp/debt-collection.test.ts

```diff
@@ -117,7 +117,7 @@ describeBspNet(
       }
 
       // Wait for tasks to execute and for the BSPs to submit proofs.
-      await sleep(500);
+      await sleep(2500);
```

Comment:
I assume this is because it failed sometimes in the CI?

Reply:
indeed, these runners are very slow